src/Client/index.html

```diff
@@ -9,8 +9,8 @@
 		<link rel="shortcut icon" type="image/png" href="/images/safe_favicon.png"/>
 
 		<!-- React -->
-		<script src="https://cdnjs.cloudflare.com/ajax/libs/react/15.4.0/react.min.js"></script>
-		<script src="https://cdnjs.cloudflare.com/ajax/libs/react/15.4.0/react-dom.min.js"></script>	
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/react/16.0.0/react.min.js"></script>
```

Comment:
@alfonsogarciacaro @MangelMaxime what's the fablish/webpackish way to deal with that?

Reply:
@forki You should not need to use react from cdn.

Your webpack.config.js doesn't list `react` as an external dependencies so it's should already be inside your `bundle.js`.

Reply:
thanks. is that the correct way?

Reply:
I would say yes.


If you want to benefit from cdn cache on your application you can probably explore webpack plugins like [dynamic-cdn-webpack-plugin](https://github.com/mastilver/dynamic-cdn-webpack-plugin).

Which will use CDN for your deps on production env. And will choose the same version as the one used in your package.json

But I didn't tested it.